ui-react/packages/atlasmap/src/UI/ExpressionContent.tsx

```diff
@@ -1,8 +1,9 @@
 import { FunctionComponent, useEffect, useCallback } from "react";
 import React from "react";
 import { Tooltip, Form, FormGroup } from "@patternfly/react-core";
-import { Subscription } from "rxjs";
+import { Subscription, Observable } from "rxjs";
 import { ExpressionFieldSearch } from "./ExpressionFieldSearch";
+import { ITextNode, IExpressionUpdatedEvent } from "../../src/Atlasmap";
```

Comment:
Can't import from `Atlasmap` from the `UI` folder. I mean, you can, but it's reversed. Components in the `UI` are meant to be decoupled from `Atlasmap` and reusable.

You can fix this by having the models live here instead than in the `AtlasmapProvider`, since this is the only place that uses them.

Reply:
Also no need to go that deep with the import, `../Atlasmap` would be enough.

Reply:
Essentially just move ITextNode and IExpressionUpdatedEvent interfaces to ExpressionContent since that's where they're needed?